source/conf.py

```diff
@@ -0,0 +1,154 @@
+from __future__ import annotations
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import sys
+from pathlib import Path
+
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+sys.path.insert(0, str(Path(__file__).parent.resolve()))
+needs_sphinx = "6.2"
+
+
+# -- Project information -----------------------------------------------------
+
+project = "Cantera"
+copyright = "2023, Cantera Developers"
+author = "Cantera Developers"
+html_title = "Cantera"
+
+# The full version, including alpha/beta/rc tags
+version = "2.6"
+release = "2.6.0"
```

Comment:
These look out-of-date, unless they're read from somewhere else?

Reply:
Oh, I guess the copyright year needs to be updated. I believe the version info here doesn't actually get shown anywhere, and strictly speaking doesn't correspond to a specific Cantera version in any case.

Reply:
I think sphinx usually puts this in some metadata somewhere. You're right, though, that this doesn't correspond to a particular version. Maybe try deleting the versions and see if things break? If they can be deleted that seems like the best route